content/docs/guides/observability/metrics.md

```diff
@@ -214,24 +214,25 @@ annotations:
-| osm_cert_issued_time                  | Histogram |                        | Histogram to track time spent to issue xds certificate                      |
-| osm_injector_injector_sidecar_count   | Count     |                        | Total number of injector webhook calls                                      |
-| osm_injector_injector_rq_time         | Histogram | success                | Histogram to track time spent performing sidecar injection                  |
-| osm_error_err_code_count              | Count     | err_code               | Number of errcodes generated by OSM                                         |
-| osm_http_response_total               | Count     | code, method, path     | Number of HTTP responses sent                                               |
-| osm_http_response_duration            | Histogram | code, method, path     | Duration in seconds of HTTP responses sent                                  |
-| osm_feature_flag_enabled              | Gauge     | feature_flag           | Represents whether a feature flag is enabled (1) or disabled (0)            |
+| Metric                                | Type      | Labels                 | Description                                                                      |
+| ------------------------------------- | --------- | ---------------------- | -------------------------------------------------------------------------------- |
+| osm_k8s_api_event_count               | Count     | type, namespace        | Number of events received from the Kubernetes API Server                         |
+| osm_proxy_connect_count               | Gauge     |                        | Number of proxies connected to OSM controller                                    |
+| osm_proxy_reconnect_count             | Count     |                        | IngressGateway defines the certificate specification for an ingress gateway      |
+| osm_proxy_response_send_success_count | Count     |                        | Number of responses successfully sent to proxies                                 |
+| osm_proxy_response_send_error_count   | Count     |                        | Number of responses that errored when being set to proxies                       |
+| osm_proxy_config_update_time          | Histogram | resource_type, success | Histogram to track time spent for proxy configuration                            |
+| osm_proxy_broadcast_event_count       | Count     |                        | Number of ProxyBroadcast events published by the OSM controller                  |
+| osm_proxy_xds_request_count           | Count     | common_name, type      | Number of XDS requests made by proxies                                           |
+| osm_proxy_max_connections_rejected    | Count     |                        | Number of proxy connections rejected due to the configured max connections limit |
```

Comment:
This is the only new line. I had to make the description column a bit wider which is why the diff shows the whole table.